glean_parser/metrics.py

```diff
@@ -208,9 +214,20 @@ class Timespan(TimeBase):
 @dataclass
 class TimingDistribution(TimeBase):
     typename = 'timing_distribution'
-    # The following is a Gecko-specific property for using the Glean SDK
-    # with GeckoView metrics. See bug 1566356 for more context.
-    gecko_datapoint: str = ''
+
+
+class HistogramType(enum.Enum):
+    linear = 0
+    exponential = 1
+
+
+@dataclass
+class CustomDistribution(Metric):
+    typename = 'custom_distribution'
```

Comment:
This might require an update depending if you want to change the constructor in mozilla-mobile/android-components#4041